src/pages/api/og/nft/[tokenId].tsx

```diff
@@ -1,10 +1,13 @@
 /* eslint-disable @next/next/no-img-element */
 import { ImageResponse } from '@vercel/og';
 import { fetchGraphql, getPreviewUrl } from '../../../../fetch';
-import { removeMarkdownStyling } from '../../../../utils/removeMarkdownStyling';
 import { tokenIdOpengraphQuery } from '../../../../queries/tokenIdOpengraphQuery';
 import { NextApiRequest } from 'next';
-import { truncateAndStripMarkdown } from '../../../../utils/extractWordsWithinLimit';
+import {
+  CHAR_LENGTH_ONE_LINE,
+    CHAR_LENGTH_TWO_LINE,
+  truncateAndStripMarkdown,
```

Comment:
there a linting issue here on the tab?

also `CHAR_LENGTH_ONE_LINE` being used in this file?